.github/workflows/dart.yml

```diff
@@ -2,16 +2,24 @@ name: Dart CI
 
 on: [push]
 
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    container:
-      image: cirrusci/flutter:v1.12.13-hotfix.8
+env:
+  DEVELOPER_DIR: /Applications/Xcode_11.3.1.app/Contents/Developer
 
+jobs:
+  test:
+    name: Test on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-16.04, macos-10.15]
```

Comment:
I guess in the future we would also want to test across certain stable flutter versions, but that can also be done in a follow-up PR